src/controller/CHIPDeviceController.cpp

```diff
@@ -1724,13 +1726,49 @@ void DeviceCommissioner::OnDone()
     CommissioningStageComplete(return_err, report);
 }
 
+// TODO if we want to branch out the Commissioning Error to a specific chip error this will be
+// moved to the top and likely is better as a map lookup (which I would look for suggestions).
+CHIP_ERROR ConvertCommissioningErrorToChipError(GeneralCommissioning::CommissioningError commissioningError)
+{
+    switch (commissioningError)
+    {
+    case GeneralCommissioning::CommissioningError::kOk:
+        return CHIP_NO_ERROR;
+        break;
+    case GeneralCommissioning::CommissioningError::kValueOutsideRange:
+        return CHIP_ERROR_INVALID_ARGUMENT; //  TODO figure out more appropriate error.
+        break;
+    case GeneralCommissioning::CommissioningError::kInvalidAuthentication:
+        return CHIP_ERROR_INVALID_ARGUMENT; //  TODO figure out more appropriate error.
+        break;
+    case GeneralCommissioning::CommissioningError::kNoFailSafe:
+        return CHIP_ERROR_INVALID_ARGUMENT; //  TODO figure out more appropriate error.
+        break;
+    case GeneralCommissioning::CommissioningError::kBusyWithOtherAdmin:
+        return CHIP_ERROR_INVALID_ARGUMENT; //  TODO figure out more appropriate error.
+        break;
+    default:
+        break;
+    }
+    return CHIP_ERROR_INVALID_ARGUMENT; //  TODO figure out more appropriate error.
```

Comment:
CHIP_ERROR_INTERNAL, maybe? I wouldn't worry too much - the whole point of the cluster-specific enums is that we don't always have enough detail in the general error codes. So "close enough" is fine here.

Reply:
Removed this mapping, no longer applicable